ci/code_checks.sh

```diff
@@ -241,8 +241,8 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
-    MSG='Validate docstrings (GL06, GL07, GL09, SS04, PR03, PR05, PR10, EX04, RT04, RT05, SS05, SA05)' ; echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=azure --errors=GL06,GL07,GL09,SS04,PR03,PR04,PR05,EX04,RT04,RT05,SS05,SA05
+    MSG='Validate docstrings (GL06, GL07, GL09, SS04, SS05, PR03, PR04, PR05, EX04, RT04, RT05, SA05)' ; echo $MSG
```

Comment:
Was there a reason for removing `PR10` here or just mistake?

Reply:
It is just a mistake from merge.

Reply:
`PR10` is actually not included in the current check at line 245. So I wan't sure someone exclude `PR10` but forgot to remove from the message.

Reply:
Good catch. I think this was just missed in #25109 so makes sense to update here accordingly

Reply:
>##[error]pandas/io/excel/_base.py(492,): error PR10: pandas.ExcelWriter: Parameter ".. versionadded" requires a space before the colon separating the parameter name and type

PR10 is not passing the check with `.. versionadded:: 0.24.`. I am going to leave PR10 for now.